modules/client-configuration/pages/clients-sle.adoc

```diff
@@ -26,10 +26,38 @@ The products you need for this procedure are:
 | {sles} 15 SP2
 | SUSE Linux Enterprise Server 15 SP2 x86_64
 
+| {sles} 15 SP3
```

Comment:
Why are you adding this?

Reply:
Because when I looked on the reference server it's there. AFAIK, we added support for the Alpha in the last MU.